src/providers/realtime-provider.js

```diff
@@ -202,6 +224,16 @@ export default class RealtimeProvider {
                         };
                         let value = getValue(parameter, parentName);
 
+                        if (this.observingStaleness[subscriptionDetails.name]) {
```

Comment:
At first glance I'm not sure whether to expect an object or a boolean here, so let's be explicit -
```
if (this.observingStaleness[subscriptionDetails.name] !== undefined)
```

Reply:
That's a great point! I'll update it.